examples/examples/staking_details.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019-2022 Parity Technologies (UK) Ltd.
+// This file is part of subxt.
+//
+// subxt is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// subxt is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with subxt.  If not, see <http://www.gnu.org/licenses/>.
+
+//! To run this example, a local polkadot node should be running. Example verified against polkadot 0.9.13-82616422d0-aarch64-macos.
```

Comment:
Question: Is this "Example verified against $version" correct for you? I'd either remove it or, better, use `polkadot --version` to put whatever version you're using in (if it's different) :)

Reply:
Or, looking at the below, perhaps the exampel is using a `substrate` instance rather than a `polkadot` one (which the other examples are using)? I'd prolly stick to `polkadot` just to be the same as the other examples (and consistent with the comments) :)